nibabel/tests/test_fileslice.py

```diff
@@ -639,8 +639,14 @@ def test_strided_scalar():
         assert_equal(observed.shape, shape)
         assert_equal(observed.dtype, expected.dtype)
         assert_array_equal(observed.strides, 0)
-        observed[..., 0] = 99
-        assert_array_equal(observed, expected * 0 + 99)
+        # Strided scalars are set as not writeable
+        # This addresses a numpy 1.10 breakage of broadcasting a strided
+        # array without resizing (see GitHub PR #358)
+        assert_false(x.flags.writeable)
```

Comment:
@effigies this line is causing a test failure; `x` is not defined. Perhaps this should be inside `setval`?